Mathlib/Combinatorics/SimpleGraph/Clique.lean

```diff
@@ -281,6 +281,11 @@ theorem CliqueFree.anti (h : G ≤ H) : H.CliqueFree n → G.CliqueFree n :=
   forall_imp fun _ ↦ mt <| IsNClique.mono h
 #align simple_graph.clique_free.anti SimpleGraph.CliqueFree.anti
 
+/-- If a graph is cliquefree, any graph that embeds into it is also cliquefree. -/
+theorem CliqueFree.map {H : SimpleGraph β} (f : H ↪g G) : G.CliqueFree n → H.CliqueFree n := by
```

Comment:
Usually `map` moves stuff from the domain to the codomain. Should it be `comap`? CC: @YaelDillies

Reply:
Otherwise LGTM but I rarely use this part of the library
bors d=@YaelDillies